Sprint-3/revise/implement/get-ordinal-number.test.js

```diff
@@ -1,4 +1,85 @@
-// In this week's prep, we started implementing getOrdinalNumber
+const getOrdinalNumber = require('./getOrdinalNumber'); // Adjust path as needed
```

Comment:
Tell me, why does this comment say adjust path as needed? 

Could this code run? Have you run it?

Reply:
The assert module is used because it's built into Node.js, simple, and effective for basic testing. It provides easy-to-use assertion methods like assert.strictEqual() to check if values are as expected.

Reply:
The comment // Adjust path as needed means I  might need to update the file path if getOrdinalNumber.js is in a different directory. The code won't run unless the file exists at the specified path.